subsys/net/lib/lwm2m/lwm2m_rd_client.c

```diff
@@ -837,7 +844,7 @@ static int sm_do_bootstrap_reg(void)
 void engine_bootstrap_finish(void)
 {
 	LOG_INF("Bootstrap data transfer done!");
-	set_sm_state(ENGINE_BOOTSTRAP_TRANS_DONE);
+	set_sm_state_delayed(ENGINE_BOOTSTRAP_TRANS_DONE, 1000);
```

Comment:
Shall we add a short comment here why this particular state transition is delayed? It might not be obvious and tracking the change commit to get an explanation might be cumbersome.

Reply:
Sure.
Added a comment.